svn/common.py

```diff
@@ -49,10 +49,13 @@ def run_command(self, subcommand, args, success_code=0,
 
         _logger.debug("RUN: %s" % (cmd,))
 
+        environment_variables = os.environ
```

Comment:
@dsoprea : It would get changed because in python assignment operator is essentially naming operator. 
```

>>> a = {'1': 2}
>>> b = a
>>> a['1'] = 3
>>> b
{'1': 3}
```